environment.yml

```diff
@@ -1,15 +1,89 @@
 name: iooswkshp
 channels:
-- conda-forge
+  - conda-forge
 dependencies:
-- python=3.6
-- requests
-- scipy
-- xarray
-- folium
-- geopandas
-- gdal=2.2
-# Jupyter notebook
-- ipykernel
-- jupyter
-- nb_conda_kernels
+  - python=3.6
+  - erddapy
+  - folium
+  - gdal=2.2
+  - geopandas
+  - requests
+  - scipy
+  - xarray
+  # Jupyter notebook
+  - ipykernel
+  - jupyter
+  - nb_conda_kernels
+  - xlrd
+  # R packages
```

Comment:
Hmm... Should these really be in the same environment file. I am using this file to build the environment for Docker. I am afraid it'll get to bulky, and maybe confusing to some people, which R to use? Are you going to have R demos in jupyter notebooks?

Reply:
> Hmm... Should these really be in the same environment file. I am using this file to build the environment for Docker. I

Like I said above I believe you are better off with the Dockerfile configuration you have right now for jupyterhub. So no, you should not use this for docker. I thought that you where going with the `requirements.txt` solution but since you are using this for docker I'll have to copy the changes here into another env file and update the instructions.

>  I am afraid it'll get to bulky, 

It will get as bulky as any installation of those packages, so if that is what people need there is no escaping that.

> and maybe confusing to some people, which R to use? Are you going to have R demos in jupyter notebooks?

I thought that single environment and a single installation instruction was the goal for non-jupyterhub and non-R savyy users, no?

Reply:
> Are you going to have R demos in jupyter notebooks?

That is an option but `rstudio` is install in that env too.

Reply:
Sorry if I confused you. 😞 

> I thought that single environment and a single installation instruction was the goal for non-jupyterhub and non-R savyy users, no?

Oh I see. That makes sense. So should I have an RKernel within the Jupyter Notebook in Jupyterhub too, in addition to RStudio?

Reply:
> That is an option but rstudio is install in that env too.

So are you saying this environment file is more for local setup?

Reply:
> So should I have an RKernel within the Jupyter Notebook in Jupyterhub too

Only if you choose to use this env file in jupyterhub, which I do not recommend b/c we don't have enough time to test all the packages.

This env file is, IMO, a plan B for people that want to install things on their own laptop, to leave the workshop with an working environment to practice, and are not R-savvy.

Reply:
> So are you saying this environment file is more for local setup?

That is what I meant when I said that you should not use this for Jupyterhub :smile: 

I'll change the file name and the instructions soon (busy working on another front at the moment),

PS: in theory we could use this in jupyterhub too but we should not change something that is working for something that is untested.

Reply:
Oh okay! Cool. I'm all clear now. I will create another file that's for the JupyterHub setup. I just gotta make sure to check often for changes in the Python packages within this Environment file. Thanks.